setup.py

```diff
@@ -37,7 +37,13 @@
         'six',
         'venusian>=1.0',
     ],
-    extra_requires={
+    extras_require={
         'Swagger': ['pyramid_swagger >= 2.3.0'],
```

Comment:
I know you didn't change this, but usually the requirement has the same name as the package if you install just one package. Also using capital letters is quite uncommon. If we ever do a major release we could change this. ;)

Reply:
As `extra_requires` was mistyped we could change it even immediately as it won't change the behaviour.
Additionally as we'll rename it to `pyramid_swagger` anyone that was installing the tool as `pyramid-mock-server[Swagger]` will continue to not install the `pyramid-swagger` dependency